packages/loader/container-loader/src/container.ts

```diff
@@ -1872,6 +1872,6 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     // Please avoid calling it directly.
     // raiseContainerWarning() is the right flow for most cases
     private logContainerError(warning: ContainerWarning) {
-        this.logger.sendErrorEvent({ eventName: "ContainerWarning" }, warning);
+        this.logger.sendErrorEvent({ eventName: "ContainerWarning", runtimeVersion: pkgVersion }, warning);
```

Comment:
this version is not the version of the runtime, this is the version of the  loader package. 

Once this https://github.com/microsoft/FluidFramework/pull/6764 is in. we should call normalizeError and annotate it with the runtime version before passing it to the container context. something similar should also be done in the close case.

That will also only handle this specific property. I think it may be possible to get a more general solution here where we can get the loggers default properties somehow, and add them to the error. that way if these properties change, we will always have them

Reply:
Thanks so much. It is better to add this issue in the implementation of #6924. Let me search for more issue to do during the FHL week.